denm_schema.js

```diff
@@ -1,177 +1,300 @@
-              "maximum": 1830297600000
+            "version": {
+                "type": "string",
+                "examples": ["0.1.0", "1.0.0"],
+                "description": "json message format version"
             },
-            "event_position": {
-              "type": "object",
-              "description": "Geographical position of the detected event.",
-              "required": ["latitude", "longitude"],
-              "properties": {
-                "latitude": {
-                  "type": "integer",
-                  "description": "latitude in microdegrees",
-                  "minimum": -900000000,
-                  "maximum": 900000001
-                },
-                "longitude": {
-                  "type": "integer",
-                  "description": "longitude in microdegrees",
-                  "minimum": -1800000000,
-                  "maximum": 1800000001
-                }
-              }
+            "source_uuid": {
+                "type": "string",
+                "examples": ["UNKNOWN"]
             },
-            "station_type": {
-              "description": "unknown(0), pedestrian(1), cyclist(2), moped(3), motorcycle(4), passengerCar(5), bus(6), lightTruck(7), heavyTruck(8), trailer(9), specialVehicles(10), tram(11), roadSideUnit(15)",
-              "type": "integer",
-              "minimum": 0,
-              "maximum": 255
-            }
-          }
-        },
-        "situation_container": {
-          "type": "object",
-          "required": ["event_type"],
-          "properties": {
-            "event_type": {
-              "type": "object",
-              "required": ["cause", "subcause"],
-              "properties": {
-                "cause": {
-                  "type": "integer",
-                  "description": "reserved (0), trafficCondition (1), accident (2), roadworks (3), impassability (5), adverseWeatherCondition-Adhesion (6), aquaplannning (7), hazardousLocation-SurfaceCondition (9), hazardousLocation-ObstacleOnTheRoad (10), hazardousLocation-AnimalOnTheRoad (11), humanPresenceOnTheRoad (12), wrongWayDriving (14), rescueAndRecoveryWorkInProgress (15), adverseWeatherCondition-ExtremeWeatherCondition (17), adverseWeatherCondition-Visibility (18), adverseWeatherCondition-Precipitation (19), slowVehicle (26), dangerousEndOfQueue (27), vehicleBreakdown (91), postCrash (92), humanProblem (93), stationaryVehicle (94), emergencyVehicleApproaching (95), hazardousLocation-DangerousCurve (96), collisionRisk (97), signalViolation (98), dangerousSituation (99)",
-                  "minimum": 0,
-                  "maximum": 255
-                },
-                "subcause": {
-                  "type": "integer",
-                  "minimum": 0,
-                  "maximum": 255
+            "timestamp": {
+                "type": "integer",
+                "description": "Unit: millisecond. The timestamp when the message was generated since Unix Epoch (1970/01/01)",
```

Comment:
we keep the classical timestamp (from 1970) here?

Reply:
Yes as agreed with Mathieu